### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MaksymMaliuk**

### CODE REVIEW

**Review by Lilu580 — COMMENTED** [review]
gj

**Review by Andr1yK — CHANGES_REQUESTED** [review]
What is 20, 50?
Avoid magic numbers

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,19 @@
...
+
+  if (days >= 7) {
+    return days * price - discountFirst;
+  }
+
+  if (days >= 3) {
+    return days * price - discountSecond;
+  } else {
+    return days * price;
+  }
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:43]: check by checklist

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountFirst = 50;
+  const discountSecond = 20;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:33]: Why first and second? where do I need to use first?

**Review by Andr1yK — CHANGES_REQUESTED** [review]

**Pull request closed without merging**